qemu/tests/fio_windows.py

```diff
@@ -23,41 +21,55 @@ def run(test, params, env):
     """
 
     install_path = params["install_path"].rstrip("\\")
-    log_file = params.get("log_file")
-    fio_cmd = params.get("fio_cmd")
+    fio_log_file = params.objects("fio_log_file")
+    fio_file_name = params.objects("fio_file_name")
+    fio_cmd_sys = params.get("fio_cmd") % (fio_file_name[0], fio_log_file[0])
+    fio_cmd_data = params.get("fio_cmd") % (fio_file_name[1], fio_log_file[1])
     timeout = float(params.get("login_timeout", 360))
-    cmd_timeout = int(params.get("cmd_timeout", "360"))
+    cmd_timeout = int(params.get("cmd_timeout", 900))
```

Comment:
This should be `fio_run_time + 60`, right?

Reply:
xuhan, it is fio_run_time + 60 before, but now, we have 2 InterruptedThreads are running, the fio test cannot complete on that time, so increase the timeout time~